web/edit/server/index.php

```diff
@@ -526,6 +526,17 @@
         }
     }
 
+    // Change login style
+    if (empty($_SESSION['error_msg'])) {
+        if ($_POST['v_login_style'] != $_SESSION['LOGIN_STYLE']) {
+            exec (HESTIA_CMD."v-change-sys-config-value LOGIN_STYLE ".escapeshellarg($_POST['v_login_style']), $output, $return_var);
+            check_return_code($return_var,$output);
+            unset($output);
+            if (empty($_SESSION['error_msg'])) $v_backup_gzip = $_POST['v_login_style'];
```

Comment:
IMHO instead of `$v_backup_gzip`  it should be `$v_login_style`

Reply:
Created a new pull request for it with the change